src/main/java/com/github/jenkinsci/zaproxyplugin/ZAPDriver.java

```diff
@@ -1265,8 +1266,8 @@ private String setUpUser(BuildListener listener, ClientApi zapClientAPI, String
         // means that any value we add for the configuration values has to be
         // URL encoded.
         StringBuilder userAuthConfig = new StringBuilder();
-        userAuthConfig.append("username=").append(URLEncoder.encode(username, "UTF-8"));
-        userAuthConfig.append("&password=").append(URLEncoder.encode(password, "UTF-8"));
+        userAuthConfig.append("Username=").append(URLEncoder.encode(username, "UTF-8"));
```

Comment:
Incorrect change i believe, you can verify which parameters are used with:

`http://zap/JSON/users/view/getAuthenticationCredentialsConfigParams/?contextId=1`

In my case it's `{"methodConfigParams":[{"name":"username","mandatory":"true"},{"name":"password","mandatory":"true"}]}`, which is also the same for Script-based.